src/bot.py

```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
     """
     await custom_meme.zoomcam(ctx, 'memes/barrington/zoombarr.png', 'zoombarr_final.png', args)
 
+@client.command(name='zoommarius', aliases=['streamermarius', 'zoom-marr', 'zoom-mar', 'zoom-marius'])
+async def zoommarius(ctx, *args):
+    """Command to paste an image of marius from Zoom onto the given image in the top right corner
+
+       Args:
+        - ctx: context that the command occured use this to access the message and other attributes
+    """
+    await custom_meme.zoomcam(ctx, 'memes/marius/zoommarius.png', 'zoommarius_final.png', args)
+
+@client.command(name='zoommarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
+async def zoommarius(ctx, *args):
```

Comment:
```suggestion
async def zoomarun(ctx, *args):
```
duplicate function name